operator/v1/types_ingress.go

```diff
@@ -1534,7 +1534,7 @@ type IngressControllerTuningOptions struct {
 	// subsequent reload.
 	//
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(s|m))+)$
+	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(ms|s|m))+)$
```

Comment:
https://go.dev/play/p/dgWS_-goMq8 shows that `time.ParseDuration` converts "0.5s10ms10ms" to 520ms, so this seems fine.